xarray/core/datatree.py

```diff
@@ -40,17 +31,22 @@
     maybe_wrap_array,
 )
 from xarray.core.variable import Variable
-
-from . import formatting, formatting_html
-from .common import TreeAttrAccessMixin
-from .mapping import TreeIsomorphismError, check_isomorphic, map_over_subtree
-from .ops import (
+from xarray.datatree_.datatree.common import TreeAttrAccessMixin
+from xarray.datatree_.datatree.formatting import datatree_repr
+from xarray.datatree_.datatree.formatting_html import (
+    datatree_repr as datatree_repr_html,
+)
+from xarray.datatree_.datatree.mapping import (
+    TreeIsomorphismError,
+    check_isomorphic,
+    map_over_subtree,
+)
+from xarray.datatree_.datatree.ops import (
     DataTreeArithmeticMixin,
     MappedDatasetMethodsMixin,
     MappedDataWithCoords,
 )
-from .render import RenderTree
-from xarray.core.treenode import NamedNode, NodePath, Tree
+from xarray.datatree_.datatree.render import RenderTree
```

Comment:
I think you are losing some of the testing if you do that.  Right now we're still collecting and running all of the tests in datatree_.  If you pull out all of the pieces that aren't migrated you'll lose that testing.  I think we're explicit about what is merged and approved by what is not in datatree_ anymore. And none of this is visible to a user yet. Let's talk in today's meeting and you can convince me otherwise.